src/main/java/duke/command/UnmarkCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.operations.TaskList;
+
+/**
+ * Represents a subclass of Command.
+ */
+public class UnmarkCommand extends Command {
+    /**
+     * A constructor for <code>UnmarkCommand</code>.
+     *
+     * @param num the index in the arraylist to be marked.
+     */
+    public UnmarkCommand(Integer num) {
+        super(null, num, null);
+    }
+
+    /**
+     * Unmarks the task in the arraylist.
+     *
+     * @param taskList the task to be marked in the arraylist.
+     * @return the task to be printed out by GUI.
+     */
+    @Override
+    public String execute(TaskList taskList) {
+        try {
+            taskList.unmark(num);
+            String firstMessage = "Surely you aren't the imposter... right??";
+            String secondMessage = taskList.toString();
+            return firstMessage + "\n   " + secondMessage;
```

Comment:
Same as previous, would it be better to put concat under one variable and return that variable?


```suggestion
            String message = "Surely you aren't the imposter... right??" + "\n" + taskList.toString();
            return message;
```